java-checks/src/main/java/org/sonar/java/checks/spring/TransactionalMethodVisibilityCheck.java

```diff
@@ -19,14 +19,18 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
 import org.sonar.check.Rule;
+import org.sonar.plugins.java.api.DependencyVersionAware;
 import org.sonar.plugins.java.api.IssuableSubscriptionVisitor;
+import org.sonar.plugins.java.api.Version;
 import org.sonar.plugins.java.api.tree.AnnotationTree;
 import org.sonar.plugins.java.api.tree.MethodTree;
 import org.sonar.plugins.java.api.tree.Tree;
 
 @Rule(key = "S2230")
-public class TransactionalMethodVisibilityCheck extends IssuableSubscriptionVisitor {
+public class TransactionalMethodVisibilityCheck extends IssuableSubscriptionVisitor implements DependencyVersionAware {
 
   private static final List<String> proxyAnnotations = List.of(
```

Comment:
Can we take the opportunity to rename these constants with UPPERCASE_SNAKECASE? I got a little confused in `visitNode` trying to understand where theses values came from?